client/src/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -75,7 +78,7 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
             }}
             InputProps={{
               classes: { input: classes.inputs },
-              endAdornment: <Typography className={classes.forgot}>Forgot?</Typography>,
+            //   endAdornment: <Link className={classes.forgot}>Forgot?</Link>,
```

Comment:
If we don't have this functionality, we don't need to comment it. It can be added later